core/src/main/java/org/jivesoftware/spark/ui/login/MutualAuthenticationSettingsPanel.java

```diff
@@ -266,7 +266,7 @@ private void addCertificate() {
             } catch (InvalidKeySpecException | NoSuchAlgorithmException | KeyStoreException | InvalidNameException
                     | IOException e) {
 
-                JOptionPane.showMessageDialog(null, "dialog.cannot.upload.certificate");
+                JOptionPane.showMessageDialog(null, Res.getString("dialog.cannot.upload.certificate"));
```

Comment:
This works as intended, but it introduces a small issue with the i18n file. The text contains quotes (which are displayed, which is ugly):

    dialog.cannot.upload.certificate = "Cannot upload certificate file"

should be

    dialog.cannot.upload.certificate = Cannot upload certificate file

It happens in a couple of places. Please fix :)

Reply:
Fixed :D